### PR TITLE
Clean up some java docs

### PR DESCRIPTION
**Pull request by lukeeey**
Cleaned up some javadocs in the IPlayer and ServerOperator classes.

Squash and merge recommended if merged

### CODE REVIEW

File: src/main/java/org/cloudburstmc/server/player/OfflinePlayer.java [hunk 1]
```diff
@@ -147,12 +141,12 @@ public Player getPlayer() {
...
     @Override
-    public Long getFirstPlayed() {
+    public long getFirstPlayed() {
         return this.namedTag != null ? this.namedTag.getLong("firstPlayed") : null;
     }
 
     @Override
-    public Long getLastPlayed() {
+    public long getLastPlayed() {
         return this.namedTag != null ? this.namedTag.getLong("lastPlayed") : null;
```
Comment (SupremeMortal) [inline src/main/java/org/cloudburstmc/server/player/OfflinePlayer.java]: This function is still returning a null value so it doesn't seem right. Maybe we could use an Optional here?

Comment (lukeeey) [reply src/main/java/org/cloudburstmc/server/player/OfflinePlayer.java]: Oh, i didn't even spot that. Makes sense why it wasnt a primitive

**Comment by lukeeey** [comment]
Don't merge yet, gonna update some more javadocs

**Comment by lukeeey** [comment]
Done

**Review by SupremeMortal — APPROVED** [review]

**Pull request merged**